build.sh

```
@@ -20,6 +20,9 @@ configurations=("v1.2" "v1.3" "v1.4" "v1.5" "v1.6")
 dotnet restore "$solutionPath"
 
 function sync_mod() {
+    rm -r ${MOD}/Languages/SpanishLatin
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)